src/ringcentral-web-phone.js

```diff
@@ -408,13 +426,13 @@
                 session.createDialog(incomingResponse, 'UAC');
                 session.hasAnswer = true;
                 session.status = 11;
+                session.emit('active-call');
```

Comment:
Here just do `if (session.userAgent.enableQos) session.startQosStatsCollection();` and do not emit event